app/controllers/collection_drafts_controller.rb

```diff
@@ -475,7 +476,30 @@ def set_projects
   end
 
   def set_platform_types
-    @platform_types = cmr_client.get_controlled_keywords('platforms')['category'].map { |category| category['value'] }.sort
+    @platform_types = cmr_client.get_controlled_keywords('platforms')['category'].map do |category|
+      short_names = category.fetch('short_name', []).map do |short_name|
+        {
+          short_name: short_name['value'],
+          long_name: short_name.fetch('long_name', [{}]).first['value']
+        }
+      end
+
+      series_entities = category.fetch('series_entity', []).map do |se|
```

Comment:
sorry to be nitpicky - I thought we wanted slightly more descriptive argument names